### ISSUE TITLE
Need more memory to run Physical Risk Models

### ISSUE THREAD

**Issue by HeatherAck** [os-climate/os_c_data_commons#332]
Farid Radmehr and Nicholas Love are running the drought model calculations on the Jupyter virtual environment, and require processing time improvements.

Both are running models in parallel:
•	We require at least 16gb memory requests and want to maximise CPUs as the code is highly parallelised
•	Currently using the memory intensive container to maximise CPUS and meet memory needs (4 CPU, 32GB, 16Gb memory requests), this takes 3 days to run a model
•	Only one of them can use the memory intensive container (the other has to use a 1CPU, 16gb memory requests container which takes 7-8 days to run a model)
•	The container size options are limited (ideally want an option with many CPUs and 16Gb memory requests)
•	Alternatively is there an easy option to grant them access to more compute on a limited basis? (as they are under pressure to get drought done before everyone disappears in August)

They have 4 more models to go which worst case will take ~10 working days, ideally this needs to be complete by Aug 10th/11th.

**Comment by HeatherAck** [comment]
@ryanaslett @redmikhail @MightyNerdEric - can one of you take this task up as a priority?

**Comment by HeatherAck** [comment]
@redmikhail created a work around; we believe this resolves issue.  If not, please open a new issue.
